dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GithubAppCheck.java

```diff
@@ -80,6 +111,52 @@ protected boolean isGithubAppEnabledOnRepository(String fullRepoName) {
         }
     }
 
+    /**
+     * Method to verify whether the github app is installed on a repository or not, using Renovate API
+     * @param fullRepoName = The repository full name, i.e, of the format "owner/repoName". Eg: "Salesforce/dockerfile-image-update"
+     * @return True if github app is installed, false otherwise. 
+     * Reference: https://github.com/mend/renovate-ce-ee/blob/main/docs/reporting-apis.md#repo-info
+     */
+    protected boolean isGithubAppEnabledOnRepositoryWithRenovateApi(String fullRepoName) {
+        try {
+            String apiEndpoint = "https://scot-mend-renovate072.sfproxy.buildndeliver.aws-esvc1-useast2.aws.sfdc.cl/api/job/repos/%s";
+            URL url = new URL(String.format(apiEndpoint, fullRepoName));
+            HttpURLConnection conn = (HttpURLConnection) url.openConnection();
+            conn.setRequestMethod("GET");
+            conn.setRequestProperty("Accept", "application/json");
+            conn.setRequestProperty("authorization", "Bearer " + appApiToken);
```

Comment:
Can we rename this to something else? appApiToken is confusing, I thought it might have something to do with the Github APP

Reply:
> Can we rename this to something else? appApiToken is confusing, I thought it might have something to do with the Github APP

Sure, I thought Renovate was just a Github app so I did it that way. I would rename `appServerApiToken` if that works, since we try to not explicitly refer to Renovate in our variable naming convention earlier

Reply:
@AvvariSaiBharadwaj done, resolving this